Assets/Scripts/Inventory/UI/DraggingOperation.cs

```diff
@@ -167,7 +167,7 @@ public void OnEndDrag(PointerEventData eventData)
         
         bool moveOccured = false;
 
-        if (destination.slot)
+        if (destination.slot && !destination.slot.Disabled)
```

Comment:
What if you just did

`if (destination.slot && destination.slot.enabled)`

Seems to me it should achieve the same thing, and you would not need to add a new property.

Reply:
You're right it would achieve the exact same thing, it's just that when I spoke a bit more in depth with nonanon about this "enabled", it looked like it was meant for greying out the slot only. I'm changing this now and will investigate with nonanon ;)

Reply:
Indeed, but your Disabled property already relied on the .enabled flag to set its logic. So I see no reason to have it as a separate property, since we can access it directly already.

And if the slot is grayed out, it should not be used. Makes sense to me :D

Reply:
I agree, it makes sense!